ee/conftest.py

```diff
@@ -91,3 +91,8 @@ def base_test_mixin_fixture():
 @pytest.fixture
 def team(base_test_mixin_fixture):
     return base_test_mixin_fixture.team
+
+
+@pytest.fixture(autouse=True)
+def use_dummy_cache_backend(settings):
```

Comment:
I wouldn't want to do this, because I want to know in my tests if I will face caching issues in the UI.

Atleast 3x before I've caught caching issues while writing API tests because my filters were somehow borked.

This setting makes it difficult to do that 😅